pkg/service/localroomstore.go

```diff
@@ -119,7 +119,7 @@ func (p *LocalRoomStore) GetParticipant(roomName, identity string) (*livekit.Par
 	return participant, nil
 }
 
-func (p *LocalRoomStore) ListParticipants(roomName string) ([]*livekit.ParticipantInfo, error) {
+func (p *LocalRoomStore) ListParticipants(roomName string, includeHidden bool) ([]*livekit.ParticipantInfo, error) {
```

Comment:
IMO, we should not hide hidden participants from room APIs.. since they are used by admins of the room